nixos/tests/networking.nix

```diff
@@ -33,13 +33,15 @@ let
         enable = true;
         interfaces = map (n: "eth${toString n}") vlanIfs;
         extraConfig = ''
+          authoritative;
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/92638.

Reply:
Huh, I was pretty sure I tested it without the authoritative line and it didn't work. It does now though, so 🤷‍♀️.

Reply:
That pull was only merged 20 hours ago.

Reply:
Yeah, but I only started working on this ~4 hours ago using then-current master.